benchmark/camelyon/pure_substrafl/register_assets.py

```diff
@@ -27,7 +27,6 @@
 
 DEFAULT_DATASET = DatasetSpec(
     name="Camelyon",
-    type="None",
     data_opener=ASSETS_DIRECTORY / "opener.py",
     description=ASSETS_DIRECTORY / "description.md",
```

Comment:
Can you also make description field optional and make it being autogenerated from the opener by default using an API call to a LLM ? Ooooooooor a simple dict lookup from the type of `op.fake_data` would do as well^^.
Aka sthg like :
```python
op = MyOpener()
X = op.fake_X()
y = op.fake_y()
type_X = type(X)
type_y = type(y)
Header = "Careful this description was Auto-Generated by parsing the output of op.fake_X, op.fake_y and might be wrong, please make sure to upload a proper description when registering a new dataset by passing the path to a markdown file to DatasetSpec.\n"
if type_X in [pd.DataFrame, torch.Tensor, torch.Dataset, np.ndarray, Path, ...]:
    Header += f"{known_types[type_X]} dataset "
    if type_X == pd.DataFrame:
        Header += "with the following fields " + " ".join(X.columns)
else:
     Header = "Unknown type {type_X} dataset, if you want the type to be recognized make sure fake_X is working appropriately"
if type_y in [np.ndarray, torch.Tensor]:
   if len(y.shape[1:]):
       "Task scalar prediction with values in {y.unique()}"
# so forth so forth
```

Reply:
Would be soooooooo cool !

Reply:
100% that this field should be optional. Thanks for the nice ideas, this would definitely be super cool :D

Reply:
do we have a card for this?

Reply:
Now we have 😇  https://linear.app/owkin/issue/FL-1513/make-descriptionmd-optional